examples/stocks/ios/.gitignore

```diff
@@ -1,11 +1,9 @@
-.idea/
 .vagrant/
```

Comment:
It might have been documented already. What's the philosophy around these template created files? Should they be more mirror matches of the original templates (maybe more auto upgradable if it hashes to old versions of the template) or more optimized and factored out into a parent (less repetition in our repo and the parent can be hand maintained to match our template gitignores)?